hacks/print-etcd-cheatsheet

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+# Copyright 2023 SAP SE or an SAP affiliate company
```

Comment:
Change the license header to the REUSE one

Reply:
Updated